### DISCUSSION TITLE
KOBO : text-shadow in epub

### DISCUSSION THREAD

**Discussion by Fafhrd69** [koreader/koreader#11342] [Ideas]
Hello Everybody,
Thanks for what is done and will be done for KOREADER.
I'm using KOREADER on on Kobo Libra H2O.
I often the epub to make the books more agreable to the eye. And I often use the property text-shadow.
It appears that KOREADER ignores this property.

Could you implement it in a future release or is there a way to apply this property ?

Thanks for any answer

Fafhrd

**Comment by Frenzie** [comment]
It sounds like something that's quite complex to implement and I suspect it wouldn't look very good on eink. But I could be wrong? Might be worth a try to see what a PDF from Prince with text-shadow looks like, I suppose.

**Comment by poire-z** [comment]
Not supported indeed, and already requested at https://github.com/koreader/crengine/issues/503.
Looking again, it's quite complex to parse, and to implement (writing text over text, not the best for legibility :) - and it's only cosmetics that may indeed not look to nice on eInk.

**Reply by Frenzie** [reply]
The linked request does bring up an interesting point in that you *can* create "shadows" that'd look alright on eink, and of course there's also the age-old text-shadow as outline hack.
